frontend_tests/node_tests/hotkey.js

```diff
@@ -155,6 +159,7 @@ function stubbing(func_name_to_stub, test_function) {
 
     set_global('popovers', {
         actions_popped: return_false,
+        reactions_popped: return_false,
```

Comment:
Might be worth adding a bit more in the way of tests, but this can be a follow-up :)